db/version_edit.h

```diff
@@ -191,6 +191,10 @@ struct LevelFilesBrief {
   }
 };
 
+// The state of a DB at any given time is referred to as a Version.
```

Comment:
Version does not include memtables.

Reply:
Right thats indirectly already documented at the Version class. Here is the comment for Version '// A column family's version consists of the SST files owned by the column
// family at a certain point in time.'